packages/core/tsconfig.cjs.json

```diff
@@ -0,0 +1,35 @@
+{
+  "compilerOptions": {
+    "module": "CommonJS",
+    "moduleResolution": "Node",
+
+    "outDir": "dist/cjs",
+    "declarationDir": "dist/types",
+
+    "esModuleInterop": true,
+    "strict": true,
+
+    "allowJs": true,
+    "declaration": true,
+    "sourceMap": true,
+    "preserveConstEnums": true,
+    "noImplicitAny": true
+
+    // "composite": true,
+    // "incremental": true,
+    // "allowSyntheticDefaultImports": true,
+    // "downlevelIteration": true,
+    // // "esModuleInterop": true,
+    // "forceConsistentCasingInFileNames": true,
+    // "isolatedModules": true,
+    // "noFallthroughCasesInSwitch": true,
+    // "noImplicitReturns": true,
+    // "noUnusedLocals": true,
+    // "resolveJsonModule": true,
+    // "skipLibCheck": true,
+    // "useDefineForClassFields": true
+    // "baseUrl": "."
```

Comment:
there's _something_ in this block that breaks the TS and idk what 😅

Reply:
what happens when it breaks?